### DISCUSSION TITLE
Game controller

### DISCUSSION THREAD

**Discussion by MaxBrille** [pybricks/support#542] [Ideas]
I'm sure you are aware of the Android app 'BrickController2'. 
Are there any intentions to include game controller (for example 'Xbox One Controller')? 
The advantage would be that there is no need of a relay station (tablet, smartphone, etc.)!

**Comment by laurensvalk** [comment]
See category 4 in https://github.com/pybricks/support/issues/262 for some generic ideas/plans.

That isn't currently our top priority, but we will consider it in future versions.
